akka-http/src/main/scala/akka/http/javadsl/server/directives/RouteAdapter.scala

```diff
@@ -10,13 +10,15 @@ import akka.http.javadsl.model.HttpResponse
 import akka.http.impl.util.JavaMapping.Implicits._
 import akka.http.javadsl.server.{ ExceptionHandler, RejectionHandler, Route, RoutingJavaMapping }
 import RoutingJavaMapping._
+import akka.annotation.InternalApi
 import akka.http.javadsl.settings.{ ParserSettings, RoutingSettings }
 import akka.http.scaladsl
 import akka.http.scaladsl.server.RouteConcatenation._
 import akka.stream.{ Materializer, javadsl }
 import akka.stream.scaladsl.Flow
 
 /** INTERNAL API */
+@InternalApi
```

Comment:
Currently there is no `impl` package under `akka-http/src/main/scala/akka/http/`. Should I add it?